be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -227,7 +244,8 @@ void ColumnReader::_calculate_row_ranges(const std::vector<uint32_t>& page_index
 Status ColumnReader::_init_ordinal_index() {
     PagePointer pp = _meta.ordinal_index_page();
     PageHandle ph;
-    RETURN_IF_ERROR(read_page(pp, &ph));
+    OlapReaderStatistics stats;
```

Comment:
You define a OlapReaderStatistics on stack, and it will be destroyed after `_init_ordinal_index()`, not passing it out.
I don't know what it is for?

Reply:
ColumnReader is shared between ColumnIterator. OlapReaderStatistics is for one-time read and should be updated by ColumnIterator. So here there is no need to pass the OlapReaderStatistics  from reader, just pass the tmp OlapReaderStatistics to api to obey the api requirement.